dnf/cli/output.py

```diff
@@ -20,6 +20,7 @@
 from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import unicode_literals
+from dnf.cli.format import format_date, format_datetime
```

Comment:
Again, line bellow
 
from dnf.cli.format import format_number, format_time

while format_number is referenced 9 times, format_time just once. Now what?